src/status_im/contexts/chat/messenger/composer/effects.cljs

```diff
@@ -150,7 +150,7 @@
            ;; UI. This way, `on-focus` will trigger first without changing the height, after which
            ;; `on-content-size-change` will animate the height of the input based on the injected
            ;; text.
-           (js/setTimeout #(do (when @messages-list-on-layout-finished? (.focus ^js @input-ref))
+           (js/setTimeout #(do (when-not composer-just-opened? (.focus ^js @input-ref))
```

Comment:
what does just-opened mean?

Reply:
1 sec ago, 1 min ?

Reply:
Changed this. Btw, thanks for pointing this out.
Not only the name is vague here, but it is also wrong.

`composer-just-opened?` does not track the opened state of the composer, but the layout calculations of the chat screen.
So even if we reopened composer here, we were keeping its value the same.

(Probably by `composer-just-opened?`, the original author wanted to express that this function is being called before other calculations (like layout) are complete. In other words, it just opened :)